### DISCUSSION TITLE
Can't view/edit my fileson Github Desktop! (beginner)

### DISCUSSION THREAD

**Discussion by maxmisling** [community/community#45317] [New to GitHub] [answered]
Hi, so not feeling like my smartest moment right now but I'm new to Github and just downloaded Github Desktop to work in. Went fine yesterday, cloned my repository, worked on my files, everything syncing fine. Then the last thing I did last night was revert a couple of changes. I had to google how to do this and I found the 'history area'(?) to do that in. However I can't remember the details of how I did that this morning and I can't find how to get back to the regular viewing and editing files area to save my life! I've tried googling and the help docs but I think it might be so basic no one has thought to explain! I've tried looking in all the menu options available as well, and it's starting to feel a bit ridiculous to be wasting so much time on this - can anyone help?
<img width="728" alt="Screenshot_20230125_172628" src="https://user-images.githubusercontent.com/60746519/214539674-a4666159-17c0-4f61-90dc-b8c1e1c30943.png">

**Comment by jge162** [comment, marked as answer]
Hello @maxmisling seems like you did some good work so far. Nevertheless, you should be able to switch back to the regular viewing and editing files area by clicking on the "Changes" tab in the top left corner of the github application. 

you should then be able to see the changes you've made to your local repository, and you should be able to commit and push those changes to the remote repository on GitHub.

You can also navigate to the local repository folder on your computer where you cloned your repository, and open the folder in your preferred text editor too.

**Comment by maxmisling** [comment]
Thanks for your response! I think I figured out what was wrong so I'll explain in case any other beginners get mixed up. I started out editing files in the Github online development editor, then found I couldn't revert commits so I downloaded Github desktop. Because it was quite late at night and the layout and visuals look similar I got mixed up. When I went back this morning I misremembered and thought I'd been editing my code on Github desktop and couldn't work out why I couldn't get back to it. I think I'm right in thinking now that Github desktop only handles the versions stuff and I actually need an external text editor to change files?
Hope this helps someone, thanks for bearing with me!
